src/ontology/mondo-edit.obo

```diff
@@ -125911,10 +125911,12 @@ replaced_by: MONDO:0013209
 
 [Term]
 id: MONDO:0007027
-name: non-alcoholic steatohepatitis
-def: "Fatty replacement and damage to the hepatocytes not related to alcohol use. It may lead to cirrhosis and liver failure." [NCIT:P378]
-synonym: "nash - nonalcoholic Steatohepatitis" EXACT [NCIT:C84445]
-synonym: "nonalcoholic Steatohepatitis" EXACT [NCIT:C84445]
+name: metabolic dysfunction-associated steatohepatitis
+def: "Metabolic dysfunction-associated steatohepatitis (MASH, formerly known as nonalcoholic steatohepatitis or NASH) is a type of fatty liver disease. It often develops due to a metabolic disorder, such as obesity or diabetes, resulting in a toxic buildup of fat in the liver. It is the most severe form of metabolic dysfunction-associated steatotic liver disease (MASLD, formerly known as nonalcoholic fatty liver disease or NAFLD)." [https://www.verywellhealth.com/non-alcoholic-steatohepatitis-nash-5196357#toc-symptoms]
```

Comment:
We usually don't start a definition by repeating the disease name. But in this case, since we include the "formerly known" names, I think it is ok.

Reply:
I agree!